generate-types/index.js

```diff
@@ -717,7 +718,7 @@ const printError = (diagnostic) => {
 				if (!isIncluded(nameForFilter)) {
 					if (name.startsWith("_") && !name.startsWith("__")) continue;
 				}
-				if (name.startsWith("__@") && !/^__@[^@]+$/.test(name)) continue;
+				if (name.startsWith("__@") && (prop.flags & ts.TypeFlags.ESSymbolLike) === 0) continue;
```

Comment:
this skips `Symbol.toStringTag` for unknown reason.. looks like TS does not know that `Symbol.toStringTag` is "es" symbol.. 

I did not dive in, looks like we are ok with not exposing `[Symbol.toStringTag](): string`